sr_multi_moveit/fast_grasping/.setup_assistant

```diff
@@ -0,0 +1,8 @@
+moveit_setup_assistant_config:
```

Comment:
We always wanted to have the ik up to the palm, but the planner used to misbehave when doing this. Trying to update the existing config in a separate branch. We'll stick with one config only.